server/main.lua

```diff
@@ -204,16 +190,47 @@ local function format_int(number)
     return minus .. int:reverse():gsub("^,", "") .. fraction
 end
 
+-- Get all bank cards for the current player
+local function getBankCard(cid)
+    local bankCard = MySQL.Sync.fetchAll('SELECT * FROM bank_cards WHERE citizenid = ? ORDER BY record_id DESC LIMIT 1', { cid })
+    return bankCard[1]
+end
+
+-- Adds a new bank card to the database, replaces existing card if it exists
+local function addNewBankCard(citizenid, cardNumber, cardPin, cardActive, cardLocked, cardType)
+    -- The use of REPLACE will act just like INSERT if there are no results that match on the citizenid key
+    -- If there are existing results, it will replace the item with the new data
+    MySQL.Async.insert('REPLACE INTO bank_cards (`citizenid`, `cardNumber`, `cardPin`, `cardActive`, `cardLocked`, `cardType`) VALUES (?, ?, ?, ?, ?, ?)', {
+        citizenid,
+        cardNumber,
+        cardPin,
+        cardActive,
+        cardLocked,
+        cardType
+    }, function(_)
```

Comment:
This function can be removed, it is unnecessary when it doesn't execute any code

Reply:
Fixed